### PR TITLE
SCRD-3497 Add monasca, os-config steps

### PR DESCRIPTION
**Pull request by GarySmith**
Add additional steps to the replace server workflow for monasca
reconfiguration and osconfig playbooks.

### CODE REVIEW

File: src/pages/ReplaceServer/ReplaceController.js [hunk 1]
```diff
@@ -190,6 +200,18 @@ class ReplaceController extends BaseUpdateWizardPage {
...
     }
 
+    playbook_steps.push(
+      {
+        label: translate('server.deploy.progress.monasca-rebuild'),
+        playbook: 'monasca-rebuild-pretasks.yml'
+      },
+      {
+        label: translate('server.deploy.progress.os-config'),
+        playbook: 'os-config.yml',
```
Comment (guglo) [inline src/pages/ReplaceServer/ReplaceController.js]: playbook name should be osconfig-run.yml

File: src/pages/ReplaceServer/ReplaceController.js [hunk 2]
```diff
@@ -38,6 +38,13 @@ class ReplaceController extends BaseUpdateWizardPage {
 
   setNextButtonDisabled = () => this.state.overallStatus != STATUS.COMPLETE;
 
+  getHostname = (internalModel, serverId) => {
+    const matches = internalModel.internal.servers.filter(s => s.id == serverId).map(s => s.hostname);
```
Comment (guglo) [inline src/pages/ReplaceServer/ReplaceController.js]: Based on my experience of adding compute,   if you need set --limit,  use day0-cp1-c1-m1 which is in server['ardana_ansible_host']. server['hostname'] gives  something like day0-cp1-c1-m1-mgmt. I never run osconfig-run.yml ...you might want to run it to see what it takes for limit.

Comment (GarySmith) [reply src/pages/ReplaceServer/ReplaceController.js]: The operations guide says it is supposed to be server['hostname'], the one with the -mgmt extension.

File: src/pages/ReplaceServer/ReplaceController.js (hunk 1 above)
Comment (GarySmith) [reply src/pages/ReplaceServer/ReplaceController.js]: Thanks, good catch

File: src/pages/ReplaceServer/ReplaceController.js (hunk 2 above)
Comment (guglo) [reply src/pages/ReplaceServer/ReplaceController.js]: if you checked, that sounds good. It is kind of messy that ansible playbook limit sometimes takes hostname with -mgmt and sometimes without.

**Pull request merged**